testing/environments/latest.yml

```diff
@@ -3,7 +3,7 @@
 version: '2.3'
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:8.9.2
```

Comment:
How is this change related?

Reply:
Found the notes on Slack, this is to get the tests green.

Reply:
Yes, this is to revert fix an incorrect automation change. It is still not passing though, so there are deeper issues.